addons/website_slides/models/slide_question.py

```diff
@@ -23,10 +25,10 @@ class SlideQuestion(models.Model):
     @api.constrains('answer_ids')
```

Comment:
Isn't this constraint on `correct_answers` and `wrong_answers`?

Reply:
option 2 :+1: